tests/test-cases/Group0-Bugs/7137.robot

```diff
@@ -33,7 +33,7 @@ Check for die events when forcing update via state refresh
     Should Be Equal As Integers  ${rc}  0
 
     # tight loop on inspect - this will force an inline state refresh
-    Run  end=$(($(date +%s) + 6));while [ $(date +%s) -lt $end ]; do docker %{VCH-PARAMS} inspect ${id} >/dev/null; done
+    Run  end=$(($(date +%s) + 12));while [ $(date +%s) -lt $end ]; do docker %{VCH-PARAMS} inspect ${id} >/dev/null; done
```

Comment:
If this was failing it's liable to be host sync latency (20s interval). As such this should be updated to be > 23.

Reply:
It has consistently passed with this change, but we can change it to 24

Reply:
Done